src/main/java/com/epam/spring/homework1/other/OtherBeanC.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.spring.homework1.other;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class OtherBeanC {
+
+    @Autowired
+    private OtherBeanC beanC;
```

Comment:
Hi Viktor. According to the homework task, here need to be injected 'BeanC'. Please correct. Thank you!